modules/ztp-image-based-upgrade-installing-lifecycle-agent-using-gitops.adoc

```diff
@@ -0,0 +1,297 @@
+// Module included in the following assemblies:
+// * scalability_and_performance/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="zp-image-based-upgrade-installing-operators-with-gitops_{context}"]
+= Installing {lcao} and the OADP Operator with ZTP GitOps
+
+You need both the {lcao} and the OADP Operator to do an image-based upgrade with ZTP GitOps.
+
+[id="zp-image-based-upgrade-installing-lcao-with-gitops_{context}"]
+== Installing the {lcao} with GitOps ZTP
+
+Install the {lcao} with GitOps ZTP to do an image-based upgrade.
+
+.Prerequisites
+
+* Create a directory called `source-crs` in the same location as the `kustomization.yaml`.
+
+.Procedure
+
+. Create the following CRs in the `openshift-adp` namespace and push them to the `source-crs` directory.
++
+--
+.LcaSubscriptionNS.yaml
+[source,yaml]
+----
+TODO
+----
+
+.LcaSubscriptionOperGroup.yaml
+[source,yaml]
+----
+TODO
+----
+
+.LcaSubscription.yaml
+[source,yaml]
+----
+TODO
```

Comment:
```suggestion
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: lifecycle-agent
  namespace: openshift-lifecycle-agent
  annotations:
    ran.openshift.io/ztp-deploy-wave: "2"
spec:
  channel: ""
  name: lifecycle-agent
  source: redhat-operators
  sourceNamespace: openshift-marketplace
  installPlanApproval: Manual
```
cc: @Missxiaoguo @rauhersu - to confirm the channel for GA

Reply:
As per Raul's comment, I added `"stable"` to `channel`